src/components/TextListItem.tsx

```diff
@@ -146,7 +148,10 @@ export const TextListItem: NamedExoticComponent<Props> & {
         ) : undefined)}
 
       {!!navigation && !withCard && (
-        <Icon.ArrowRight color={colors.darkText} size={normalize(18)} />
+        <>
+          {!!count && <BoldText>({count})</BoldText>}
```

Comment:
if we place the count on the right, it should not be surrounded by brackets

Reply:
Yes, you are right. I'll fix it. 👍

Reply:
no, i meant we can print 1 instead of (1)

Reply:
Ach sorry I misunderstood. I used `()` to make the application look the same as the category list. But you are right. If we show it on the right side, we don't need `()`.